### PR TITLE
Sp3 accelerometer

### PR DESCRIPTION
**Pull request by nkalscheuer**
Adding accelerometer implementation into data model.

### CODE REVIEW

File: binnacle/lib/ListAngleWidget.dart [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+  void updateListAngle(double listAngle) {
+    setState(() {
+      _listAngle = listAngle;
+    });
+  }
+  String listText(double radians) {
+    return "List Angle: " + format.format(radiansToDegrees(radians)) + " deg";
+  }
+
+  double radiansToDegrees(double radians) {
```
Comment (CaseyHillers) [inline binnacle/lib/ListAngleWidget.dart]: Can we move this to a utility function? e.g. `utils/Math.dart`
i'd like to use it in some of my code

File: binnacle/lib/model/phone/Boat.dart [hunk 2]
```diff
@@ -1,20 +1,54 @@
...
+      // phone's way of calculating the
+      // angle of list
+      if (event != null) {
+        listAngle =  eventToListAngle(event);
+      }
+      yield listAngle;
+    }
+  }
+
+  double eventToListAngle(AccelerometerEvent event) {
```
Comment (CaseyHillers) [inline binnacle/lib/model/phone/Boat.dart]: I think we should move this to be a function in utils incase we have to do something similar for the bluetooth sensors.

`utils/Boat.dart`?

File: binnacle/test/SpeedWidget_test.dart [hunk 3]
```diff
@@ -1,6 +1,5 @@
 import 'package:flutter_test/flutter_test.dart';
 import 'package:sos/SpeedWidget.dart';
-import 'package:mockito/mockito.dart';
```
Comment (CaseyHillers) [inline binnacle/test/SpeedWidget_test.dart]: Why was this removed?

Comment (nkalscheuer) [reply binnacle/test/SpeedWidget_test.dart]: it was an unused import
